packages/compiler-core/src/transforms/transformElement.ts

```diff
@@ -125,6 +125,13 @@ export const transformElement: NodeTransform = (node, context) => {
         args.push(propsBuildResult.props)
       }
     }
+
+    if (node.tagType === ElementTypes.PORTAL && !findProp(node, 'target')) {
```

Comment:
I know this is an old PR, but given it's still open...

Should this maybe take account of `v-bind="obj"` too? The target could be passed in `obj` rather than as a separate prop.

Reply:
Agreed with @skirtles-code. We also need to consider `v-bind="obj"`, which cannot be done at compile phase. So we should do it at runtime. see
https://github.com/vuejs/core/blob/928af5fe2f5f366b5c28b8549c3728735c8d8318/packages/runtime-core/src/components/Teleport.ts#L56-L63

I think this PR can be closed. @skirtles-code WDYT?